src/storage/src/hummock/compactor/mod.rs

```diff
@@ -73,6 +75,7 @@ use crate::hummock::{
 pub struct Compactor {
     /// The context of the compactor.
     context: Arc<CompactorContext>,
+    sstable_object_id_manager: Box<dyn GetObjectId>,
```

Comment:
This is already part of the CompactorContext. Why do we need to store it again here?